node/resolvers/Mutations/Organizations.ts

```diff
@@ -307,6 +307,12 @@ const createOrganizationAndCostCenterWithAdminUser = async (
       })
     }
 
+    message({
```

Comment:
Do you think you should check the settings here and keep with up with the pattern for the other mutations?

Something like:


```suggestion
if(settings?.transactionEmailSettings?.organizationCreated) {
    message({
          logger,
          mail,
      storefrontPermissions,
    }).organizationCreated(organizationInput.name)
}
```